tests/GIN_converter_spec.yml

```diff
@@ -0,0 +1,47 @@
+global_metadata:
+  NWBFile:
+    lab: My Lab
+    institution: My Institution
+
+experiments:
+  ymaze:
+    experiment_metadata:
+      NWBFile:
+        session_description: Subject navigating a Y-shaped maze.
+    data_interfaces:
+      - SpikeGLXRecordingInterface
+
+    sessions:
+      - nwbfile_path: example_converter_spec_1.nwb
+        source_data:
+          SpikeGLXRecordingInterface:
+            file_path: ephy_testing_data/spikeglx/Noise4Sam_g0/Noise4Sam_g0_imec0/Noise4Sam_g0_t0.imec0.ap.bin
+        session_metadata:
+          NWBFile:
+            session_start_time: "2020-11-09T21:19:09+00:00"
+          Subject:
+            subject_id: "001"
+      - nwbfile_path: example_converter_spec_2.nwb
+        session_metadata:
+          NWBFile:
+            session_start_time: "2020-11-10T21:19:09+00:00"
+          Subject:
+            subject_id: "002"
+        source_data:
+          example_recording:
+          example_sorting:
```

Comment:
what are these for?

Reply:
Haven't polished this one yet - these are going to be replaced by actual file paths to actual data from the GIN testing repo, whereas the other one uses the tutorial interfaces